src/SonarScanner.MSBuild.Common/Resources.resx

```diff
@@ -266,4 +266,7 @@
   <data name="WARN_FailedToReadFile" xml:space="preserve">
     <value>Cannot detect the operating system. {0}</value>
   </data>
+  <data name="MSG_AnalysisFileCouldNotBeAdded" xml:space="preserve">
+    <value>File could not be added to analysis files: '{0}'.</value>
```

Comment:
What do you think about:
"Could not add `{0}` to the analysis."
and the exception can be added when debug mode is active. (separate line).

Reply:
This is also a debug-only log, as I do not think it is important enough to log it as an info.
But I prefer your format more, so I will change the message.